ballot/admin/ballot.py

```diff
@@ -2,4 +2,9 @@
 
 from ..models import ballot as models
 
-admin.site.register(models.Ballot)
+
+class BallotAdmin(admin.ModelAdmin):
+    exclude = ['locality']
```

Comment:
So every zipcode has its on ballot???  I thought a ballot was attached to a jurisdiction, e.g. Oakland.  What values do you want to filter out of the zip code drop down?  It should have cities and counties in it I think.